lib/vmfloaty.rb

```diff
@@ -16,7 +16,7 @@ class Vmfloaty
   include Commander::Methods
 
   def run
-    program :version, Version.get
+    program :version, Vmfloaty::VERSION
```

Comment:
Cool, haven't tried this yet but I've tried to get this to work this way for a couple of times now and it never seems to behave correctly.  👍